ipfx/stim_features.py

```diff
@@ -32,6 +32,9 @@ def get_stim_characteristics(i, t, test_pulse=True):
     else:
         amplitude = float(peak_low)
 
+    if np.isnan(amplitude):
```

Comment:
I believe it would be better to move the lines:
```
        stim_features = current_clamp_sweep_stim_features(sweep)
        sweep_features.update(stim_features)
```
inside `if not tags:` condition on line 286, then you would not need to add nan handling here.

Reply:
this would't work because stim_features (in particular stimulus amplitude) are required outputs, qc is not